packages/tabs/src/Default/Default.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Meta } from '@storybook/react';
+import { Component } from './Default';
+import descriptionMd from './Description.md';
+
+export { Default } from './stories/Default-index.stories';
+
+export default {
+  title: 'Components/Tabs/Default',
+  component: Component,
+  parameters: {
+    docs: {
+      description: {
+        component: [descriptionMd].join('/n'),
+      },
+    },
+  },
+  decorators: [
+    Story => (
+      <div style={{ display: 'block', color: 'black', width: '30%' }}>
```

Comment:
You can remove the `color` and change `width` to 50% here. The color will be set by the component and the width 30% makes the tabs break to the line below when using the 'Lg' font size.